operator/internal/central/extensions/reconcile_tls.go

```diff
@@ -142,6 +162,29 @@ func (r *createCentralTLSExtensionRun) validateAndConsumeCentralTLSData(fileMap
 	if err := r.ca.CheckProperties(); err != nil {
 		return errors.Wrap(err, "loaded service CA certificate is invalid")
 	}
+
+	// the presence of a secondary CA certificate is optional
+	secondaryCA, err := certgen.LoadSecondaryCAFromFileMap(fileMap)
+	var secondaryCACert *x509.Certificate = nil
+	if err != nil && !errors.Is(err, certgen.ErrNoCACert) {
+		return errors.Wrap(err, "loading secondary CA failed")
+	}
+	if secondaryCA != nil {
+		if err := secondaryCA.CheckProperties(); err != nil {
+			return errors.Wrap(err, "loaded secondary service CA certificate is invalid")
+		}
+		secondaryCACert = secondaryCA.Certificate()
+	}
+
+	r.caRotationAction, err = r.checkCARotation(r.ca.Certificate(), secondaryCACert)
+	if err != nil {
+		// this should only happen if the primary CA is invalid
+		return errors.Wrap(err, "checking CA rotation failed")
```

Comment:
This error message is cryptic (as an end user I would have no idea what "checking CA rotation" means) probably as a consequence of the checkCARotation method combining two purposes:
1. checking validity of the primary CA
2. determining the rotation action (if any)

I'd suggest splitting it into two.